src/components/sessions/SessionTimeline.tsx

```diff
@@ -124,12 +127,24 @@ function SessionTimeline({
   handleTogglePublicEvaluation,
   handleResetQuestionBlock,
   handleActivateBlockById,
+  handleActiveBlock,
+  handleNoActiveBlock,
   subscribeToMore,
 }: Props): React.ReactElement {
   useEffect((): void => {
     subscribeToMore()
   }, [])
 
+  const [isBlockActive, setIsBlockActive] = useState(activeStep % 2 === 1)
```

Comment:
I think you can just use a normal const for such "derived" properties that are never updated from within the component (i.e., just in your useEffect). For example

```suggestion
  const isBlockActive = activeStep % 2 === 1
```

Reply:
It will be recomputed on every render but that can be neglected for such trivial calculations.

Reply:
See comment below.